MODULE.bazel

```diff
@@ -28,6 +28,12 @@ bazel_dep(
     name = "rules_android",
     version = "0.1.1",
 )
+# Remove this once rules_android has an updated BCR release.
```

Comment:
...newer than `0.5.0.bcr.1`?

Reply:
https://github.com/bazelbuild/rules_android/commit/f4b6edb93e13ba0bd17ecf7f99c80f4d3d6ca767 is required for the windows tests to pass, which was after the 0.5.0 release

if it would be preferred we can do a 0.5.1 release with the latest code

Reply:
we made a 0.5.1 release: https://github.com/bazelbuild/rules_android/releases/tag/v0.5.1
just have to add it to BCR: https://github.com/bazelbuild/bazel-central-registry/pull/2554

Reply:
0.5.1 is available on the BCR now so I'll rerun the tests with that version